backends/ent/store.go

```diff
@@ -441,3 +467,11 @@ func (backend *Backend) newNodeCreate(n *sbom.Node) *ent.NodeCreate {
 
 	return newNode
 }
+
+func rollback(tx *ent.Tx, err error) error {
+	if rollbackErr := tx.Rollback(); rollbackErr != nil {
+		err = fmt.Errorf("rolling back transaction: %w", rollbackErr)
```

Comment:
you can return here

```suggestion
		return fmt.Errorf("rolling back transaction: %w", rollbackErr)
```

Reply:
@cpanato I applied your suggestion but it dismissed your review. Mind re-approving?